src/routes/wallet/Send.svelte

```diff
@@ -117,6 +115,16 @@
 					break;
 				case AssetType.WTN:
 					{
+						if (maybeAccount instanceof AccountIdentifier) {
+							toasts.add(
+								Toast.error(
+									'Transfer failed: wtn transfers require a principal. Please provide a valid principal.'
+								)
+							);
+							isSending = false;
+							inSendingMenu.set(false);
+							return;
```

Comment:
You close the isSendingMenu if the destination is not correct?

Reply:
Yes because the sending container is at index 1 and is causing the blur on all of the other content. If I don't close it's ugly and would require changing a few other things to make it work. I can add that to the roadmap if you want to